### PR TITLE
(client): TeamRow component finished

### PR DESCRIPTION
**Pull request by MrFiddle**
### Components
- `TeamRow.tsx`
    - Row part of a future component called TeamTable.
  
- `types.ts`
    - Attribute added to 'User' interface.
 
- `MyTeamScreen.tsx`
    - Temporal dummy data added in top of component to render the TeamRow component.
    - Change Header title to 'My Teams' instead of 'My Team'.  

- `CertificationRow.tsx`
    - Change items position to align them with the top bar of the table.
 
- `Header.tsx`
    - Changed font-weight attribute to the text of it to a heavier one. (ik it's not part of what I had to do but me dio fomo srry)  

### Description
- `TeamRow.tsx` component created, it renders a card that contains an employee, with its name, role and a button to visit their profile.

## Test Plan
Captura de pantalla 2023-04-24 130607

### CODE REVIEW

File: client/lib/types.ts
Comment (EvilDao-RG) [inline client/lib/types.ts:1]: Este atributo no rompe el componente del sidebar? Mejor crea otro tipo, que sea Employee

**Review by EvilDao-RG — CHANGES_REQUESTED** [review]
Regresa el tipo User a como estaba y pon otro tipo llamado employee

File: client/lib/types.ts
Comment (emiliogonmar2002) [reply client/lib/types.ts:1]: Dejarlo en el mismo interface, esa info es de todo el usuario

File: client/lib/types.ts [hunk 1]
```diff
@@ -21,4 +21,5 @@ export interface User {
   name: string;
   lastName: string;
   email: string;
+  rol: string;
```
Comment (emiliogonmar2002) [inline client/lib/types.ts:24]: role

**Review by emiliogonmar2002 — APPROVED** [review]
LGTM

**Pull request merged**